docs/conf.py

```diff
@@ -56,11 +53,15 @@
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
-#
-# The short X.Y version.
-version = '4.1.0'
+
 # The full version, including alpha/beta/rc tags.
-release = '4.1.0'
+import io
+with io.open('../version.txt') as f:
+    release = f.read().strip()
```

Comment:
Ugh!  I'd really rather just use the `pkg_resources` bit to find the version number of the installed package.